tests/unit/runners/test_filequeue.py

```diff
@@ -0,0 +1,172 @@
+
+import tempfile
+import shutil
+import os
+import pytest
+import time
+from mock import patch, MagicMock
+from srv.modules.runners import filequeue
+
+class TestFileQueue():
+    '''
+    This class tests the FileQueue class for the salt runner
+    '''
+    def test_default_queue(self):
+        '''
+        Verify that the default queue exists
+        '''
+        dirpath = tempfile.mkdtemp()
```

Comment:
instead of wirting `dirpath = tempfile.mkdtemp()` in every method, pytest suggests to use _fixtures_

http://doc.pytest.org/en/latest/fixture.html